src/main/java/com/softserve/util/ScheduleConstants.java

```diff
@@ -0,0 +1,35 @@
+package com.softserve.util;
+
+public final class ScheduleConstants {
```

Comment:
I don't these are all schedule-related constants, thus "Schedule" appears a bit misleading, just Constants would be better

Reply:
Okay, I wanted to associate the class name with the project name